Source/BestProjectEver/Public/Core/GameModes/BPE_GameplayGameMode.h

```diff
@@ -17,23 +18,53 @@ class BESTPROJECTEVER_API ABPE_GameplayGameMode : public AGameMode
 {
 	GENERATED_BODY()
 
+protected:
+
+	UPROPERTY(VisibleAnywhere, BlueprintReadOnly, Category="Game Mode")
+	float TimeLeft;
+
+	UPROPERTY(EditDefaultsOnly, Category="Game Mode")
+	float MatchTime;
+	
+	UPROPERTY(EditDefaultsOnly, Category="Game Mode")
+	float WarmupTime;
+
+	float LevelStartingTime;
+
+	/** rate to update time left */
+	UPROPERTY(Transient)
```

Comment:
Timer handles have no need of being UPROPERTYs

Reply:
Thanksss I erase it and replace it with a delegate